aspnetcore/migration/fx-to-core/index.md

```diff
@@ -43,7 +43,7 @@ Many applications have cross-cutting concerns that span multiple layers and need
 Supporting libraries often have complex dependency relationships that require careful upgrade ordering:
 
 * **Dependency tree complexity** - Libraries must be upgraded in postorder depth-first search ordering
-* **Multi-targeting requirements** - Libraries need to support both .NET Framework and .NET Core/.NET Standard
+* **Multi-targeting requirements** - Libraries need to support all targeted framework versions.
```

Comment:
```suggestion
* **Multi-targeting requirements** - Libraries need to support all targeted framework versions.
```
We could put all targeted .NET versions but it's not clear that that's better than framework versions here